src/providers/twitch/TwitchChannel.cpp

```diff
@@ -37,6 +37,14 @@ namespace chatterino {
 namespace {
     constexpr int TITLE_REFRESH_PERIOD = 10;
     constexpr char MAGIC_MESSAGE_SUFFIX[] = u8" \U000E0000";
+    static const QString CLIPS_LINK("https://clips.twitch.tv/%1");
+    static const QString FAILURE_TEXT("Failed to create a clip - either the "
+                                      "streamer has clips disabled or you "
+                                      "haven't re-authenticated yet. ");
+    static const QString LOGIN_PROMPT_TEXT(
+        "Click here to add your account again.");
+    static const Link ACCOUNTS_LINK(Link::OpenAccountsPage, QString());
+    static const int CLIP_CREATION_COOLDOWN = 5000;
```

Comment:
Since these are already in the unnamed namespace, we should be able to drop the `static` keyword here